connector/mqtt/vernemq/v2k-brigde/app/config.js

```diff
@@ -1,17 +1,35 @@
+const hostName = process.env.HOSTNAME || 'v2k-bridge-verne';
+const unsecuredMode = (mode) => ((mode || false) && (mode.toString().toLowerCase().trim() === 'true' || Number(mode) > 0));
+
 const app = {
-  mqttLogLevel: process.env.LOG_LEVEL || 'info',
+  mqttLogLevel: process.env.LOG_LEVEL || 'debug',
+  baseDir: process.env.BASE_DIR || '/opt/v2k-verne',
```

Comment:
Is this necessary?

Reply:
Yes, it is necessary to keep tracking correctly where the tls files are generated.